### ISSUE TITLE
After upgrading to p2p messages are being received twice per send.

### ISSUE THREAD

**Issue by aintnorest** [aintnorest/ubiquitous-spoon#3]
server/tests/ioConnect.test.js

Test:  socket.io integration test - request game

lines 189 - 213
I can tell the test is only emitting the message once but the socket receiving it gets it twice. using a timestamp or a random number I can just check for the duplicate and delete it but not really a great solution. Thinking I'll keep my mind on it for the weekend but if I can't figure it out I may just drop socket io and implement it myself.

**Comment by aintnorest** [comment]
I did write a vanilla WebRTC implementation and tried it instead of the socket.io-p2p. I didn't receive duplicates It also didn't have an ice connection which later in the week my boss said will always send the first message twice as part of what it does to get through a firewall. I tested the socket.io-p2p again to see if it was sending more than just the first message in each direction twice, it is. I don't know if this is a result of the ICE maybe trying to get through a firewall that doesn't exist or a problem with the implementation. 

I've decided to write a proxy that will give a static api for the client and server to use so that I can keep tinkering and learning about the WebRTC connection without hindering the rest of the process. Hopefully I'll get that pushed tonight.
